File: Codeforces/379/p2.cpp
#include<iostream>
#include<cstdio>
#include<iostream>
#include<cstdio>
#include<cstring>
#include<string>
#include<vector>

using namespace std;

int main(){
    int two,three,five,six;
    scanf("%d %d %d %d",&two,&three,&five,&six);
    int sum = 0;
    while(two > 0 && five > 0 && six > 0){
        sum+=256;
        two--;five--;six--;
    }
    while(two > 0 && three > 0){
        sum+=32;
        two--;
        three--;
    }
    cout << sum << endl;
    return 0;
}

File: AIPO/2018-Prelims/p2.py
n = {'zero':0,
    'one':1,
    'two':2,
    'three':3,
    'four':4,
    'five':5,
    'six':6,
    'seven':7,
    'eight':8,
    'nine':9}
o = ['zero','one','two','three','four','five','six','seven','eight','nine','ten','eleven','twelve','thirteen','fourteen','fifteen','sixteen','seventeen','eighteen']
l = input().split()
A = n[l[0]]
B = n[l[2]]
print(o[A+B])
